template-parts/updatebox-section.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * Climate Hub partners page supplementary updatebox section template
+ * @package climatehub
+ */
+
+
+  if(get_field('enable_custom_field')): ?>
+    <div class="container">
+      <section>
+        <div class="row">
+          <div class="col-sm">
```

Comment:
The class here can be more specific. For example, on small screens we want this to stack, so `col-12 col-md-4` would do.